README.md

```diff
@@ -2,31 +2,96 @@
 
-## Getting Started
+## Quickstart Instructions
 
 ### Prerequisites
 
-1. You must have access to the Yext Github organization: https://github.com/yext
-1. You must have access to the Yext NPM organization: https://www.npmjs.com/org/yext
 1. Have the Yext CLI installed: https://hitchhikers.yext.com/guides/cli-getting-started-resources/01-install-cli/
-1. Have Deno installed, version 1.21.0 or later: https://deno.land/manual/getting_started/installation
-1. Have node installed, version 17 or later: https://nodejs.org/en/download/
+2. Have Deno installed, version 1.21.0 or later: https://deno.land/manual/getting_started/installation
+3. Have node installed, version 17 or later: https://nodejs.org/en/download/
 
    - It's recommend to use nvm: https://github.com/nvm-sh/nvm#installing-and-updating or via brew `brew install nvm`
 
-1. Optional: Have a Yext account (necessary for production builds, deploying on Yext Sites, and pulling local stream document data via `yext sites generate-test-data`). This starter already comes with some localData that can be used for local dev without the need to init with a Yext account.
-
-### Clone this repo and install dependencies
-
-```shell
-git clone git@github.com:yextconsulting/site-starter-react-consulting.git
-cd site-starter-react-consulting
-npm install
-```
-
-### Dev workflow
-
-### Useful commands
+4. Optional: Have a Yext account (necessary for production builds, deploying on Yext Sites, and pulling local stream document data via `yext sites generate-test-data`). This starter already comes with some localData that can be used for local dev without the need to init with a Yext account.
+
+### Getting started
+
+1. Clone this repo and install dependencies:
+
+   ```bash
+   git clone https://github.com/yextconsulting/pages-starter-react-consulting.git
+   cd site-starter-react-consulting
+   npm install
+   ```
+
+2. Run `yext init` and authenticate the CLI with your Yext credentials. Add the `-u sandbox` flag if you're using a sandbox Yext account.
+3. Run `yext resources apply platform-config/config` to apply the required configuration resources to your account.
+4. Run `yext resources apply platform-config/entities` to add the necessary entities to your account. This includes a test location entity which can be deleted before running the apply command if desired.
+
+   - You will be prompted to enter several API keys during this step which power the `<Reviews>`, `<Nearby>` and `<Locator>` components. If you do not plan to use these components, you can skip the API key input by pressing Enter.
+   - The Reviews API key should come from a developer app with Read access to the Management API > Reviews endpoint.
+   - For both the Nearby API key as well as the Search API key, you can use an API key from any search experience in your account or a developer app with Read access to the Content Delivery API > Entities endpoint.
```

Comment:
I didn't realize you could use any key with entities access for your search api requests? That seems like something we maybe shouldn't suggest even if it technically works

Reply:
Yea that's fair, I can separate the two and say that the Search key should come from the search experience